scripts/report_transactions.js

```diff
@@ -78,30 +78,31 @@ async.auto({
       .done(cb);
   },
 
-  newCollectiveCount: cb => {
+  newCollectives: cb => {
     models.Group
-      .count(createdLastWeek)
+      .findAll(_.merge({}, { attributes: ['slug']}, createdLastWeek))
+      .map(group => group.dataValues.slug)
       .done(cb);
   },
 
   donationAmount: cb => {
     models.Transaction
         .aggregate('amount', 'SUM', _.merge({}, lastWeekDonations, groupByCurrency))
-        .map(row => ' ' + row.SUM + ' ' + row.currency)
+        .map(row => row.SUM + ' ' + row.currency)
```

Comment:
I'm surprised eslint isn't complaining about quotes here. Can we switch to template literals pls?